src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -118,7 +118,7 @@ struct jit_uni_def_conv_kernel_f32 : public jit_uni_def_conv_kernel, public jit_
     Xbyak::Label l_table;
 
     inline void checkZeroWei(const Xbyak::Xmm &x1, Label &nullifyLabel) {
-        uni_vtestps(x1, x1);
+        ptest(x1, x1);
```

Comment:
Need confirm. @zhangYiIntel

Reply:
I check the uni_vtestps implementation, the `uni_vtestps`  only wraps ptest

```
    void uni_vtestps(const Xbyak::Xmm &x1, const Xbyak::Operand &op) {
        ptest(x1, op);
    }
```

However, the oneDNN2.6 has change the implementation which generates code we don't want.
```
    void uni_vtestps(const Xbyak::Xmm &x1, const Xbyak::Operand &op) {
        if (is_valid_isa(avx))
            vptest(x1, op);
        else
            ptest(x1, op);
    }
```